app/src/main/java/com/kickstarter/libs/Koala.java

```diff
@@ -180,26 +180,35 @@ public void trackActivityView(final int pageCount) {
 
   // SEARCH
   public void trackSearchView() {
-    client.track("Discover Search");
+    client.track(KoalaEvent.VIEWED_SEARCH);
+    client.track(KoalaEvent.DISCOVER_SEARCH_LEGACY);
   }
 
   public void trackSearchResults(final @NonNull String query, final int pageCount) {
     if (pageCount == 1) {
-      client.track("Discover Search Results", new HashMap<String, Object>() {
+      final Map<String, Object> params = new HashMap<String, Object>() {
         {
           put("search_term", query);
         }
-      });
+      };
+      client.track(KoalaEvent.LOADED_SEARCH_RESULTS, params);
+      client.track(KoalaEvent.DISCOVER_SEARCH_RESULTS_LEGACY, params);
     } else {
-      client.track("Discover Search Results Load More", new HashMap<String, Object>() {
+      final Map<String, Object> params = new HashMap<String, Object>() {
         {
           put("search_term", query);
           put("page_count", pageCount);
         }
-      });
+      };
+      client.track(KoalaEvent.LOADED_MORE_SEARCH_RESULTS, params);
+      client.track(KoalaEvent.DISCOVER_SEARCH_RESULTS_LOAD_MORE_LEGACY, params);
     }
   }
 
+  public void trackSearchCleared() {
```

Comment:
while we're at adding new tracking methods, we may as well name it closer to the event name itself e.g. `trackClearedSearchTerm()` to make them easier to find